liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PackageServerMojo.java

```diff
@@ -36,6 +36,9 @@
      */
     @Parameter(property = "packageFile")
     private File packageFile = null;
+    
+    @Parameter(property = "project.build.directory")
+    private String projectBuildDir;
```

Comment:
project has already been injected in the super class, AbstractLibertySupport
```code
    /**
     * Maven Project
     */
    @Parameter(defaultValue = "${project}", required = true, readonly = true)
    protected MavenProject project = null;
``` 
You can also set `private String projectBuildDir = project.getBuild().getDirectory();` without using parameter injection.